src/vibe/scss/components/avatars.scss

```diff
@@ -36,5 +36,23 @@
 	    }
 	}
 
-	
+	&.avatar-medium {
+	    width: 48px;
+	    height: 48px;
+
+	    .user-initials {
+	    	line-height: 48px;
+	    	font-size: 24px;
+	    }
+	}
+
+	&.avatar-big {
```

Comment:
I would recommend using `large` instead of `big`. This will align more with how bootstrap names their sizes.